frontend/src/components/Interactions/MenuItem.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+    <li class="menu-item">
+        <button class="menu-button">
+            <slot/>
+        </button>
+    </li>
+</template>
+
+<style scoped>
+.menu-item {
+    padding: 7px 0;
```

Comment:
i think you should put this padding on the button itself - currently you can click outside the button (you don't get the pointer cursor), but since you still click the `li`, the `@click.native` in the parent component triggers

Reply:
Yep good pickup.

Reply:
fixed.